source/screens/caseScreens/EKBCases.js

```diff
@@ -0,0 +1,196 @@
+    setCompletedCases(relCases.filter(c => c.status !== 'ongoing'));
+  }, [caseType, cases]);
+
+  const StatusComponent = () => {
+    switch (status) {
+      case Status.untouched:
+        return (
+          <>
+            <Heading type="h3">Status</Heading>
+            <Text>Du har ingen aktiv grundansökan. </Text>
+            <ButtonContainer>
+              <Button
+                color="green"
+                onClick={() => {
+                  createCase(basicApplicationFormId, newCase => {
+                    navigation.navigate('Form', { caseData: newCase });
+                  });
+                }}
+              >
+                <Text>Ansök om ekonomiskt bistånd</Text>
+              </Button>
+            </ButtonContainer>
+          </>
+        );
+      case Status.unfinishedNoCompleted:
+      case Status.unfinished:
+        return (
+          <>
+            <Heading type="h3">Status</Heading>
+            <Text>
+              Du har en påbörjad {latestCase.formId === recurringFormId ? 'löpande ' : 'grund'}
+              ansökan, senast uppdaterad {getFormattedUpdatedDate(latestCase)}.
+            </Text>
+            {latestCase.formId === recurringFormId && (
+              <Text>Just nu gäller ansökan perioden XX - YY. Skicka in din ansökan innan ZZ. </Text>
+            )}
+            <ButtonContainer>
+              <Button
+                color="blue"
+                onClick={() => {
+                  navigation.navigate('Form', { caseId: latestCase.id });
+                }}
+              >
+                <Text>Fortsätt ansökan</Text>
+              </Button>
+            </ButtonContainer>
+          </>
+        );
+      case Status.recentlyCompleted:
+        return (
+          <>
+            <Text>
+              Du har en inskickad {latestCase.formId === recurringFormId ? 'löpande ' : 'grund'}
+              ansökan.
+            </Text>
+            <Text>Just nu gäller ansökan perioden XX - YY. Skicka in din ansökan innan ZZ. </Text>
+            <Button
+              color="green"
+              onClick={() => {
+                createCase(recurringFormId, newCase => {
+                  navigation.navigate('Form', { caseData: newCase });
+                });
+              }}
+            >
+              <Text>Ansök om ekonomiskt bistånd för perioden XX - YY</Text>
+            </Button>
+          </>
+        );
+      default:
+        break;
+    }
+  };
+
+  const ContactInfoComponent = () => {
+    if (completedCases.length > 0) {
+      return (
+        <>
+          <Heading type="h3">Kontaktinformation</Heading>
+          <Text>
+            Om du har frågor eller behöver hjälp med något som gäller ansökan om ekonomiskt bistånd,
+            kan du ta kontakt med din handläggare.
+          </Text>
+          <Text>Vi behöver få in den information någonstans...</Text>
+        </>
+      );
+    }
+    return null;
+  };
+
+  const CompletedCasesComponent = () => {
```

Comment:
For some reason this array of cases vanished when i pressed one of the cases.

Reply:
This was because we were setting status=ongoing when closing the forms, which was a bug/mistake. This is now fixed, and I've added logic so that cases with status!==ongoing cannot be edited.

We still need a better summary view to show when the user opens a submitted case.